crates/ruff_linter/src/rules/flake8_bandit/rules/suspicious_function_call.rs

```diff
@@ -1176,7 +1179,7 @@ fn suspicious_function(
                     }
                 }
             }
-            Diagnostic::new(SuspiciousURLOpenUsage, range)
+            checker.checked_report_diagnostic(SuspiciousURLOpenUsage, range)
```

Comment:
Maybe: `report_diagnostic_if_enabled`. THe `checker.checked` reads a bit strangely